src/charts/pie-chart.js

```diff
@@ -182,15 +181,9 @@ export default function pieChart(parent, chartGroup) {
     const chartData = _chart.data()
 
     let pieData
-    if (_chart.customDomain().length > 0 && chartData) {
-      pieData = chartData.filter(d => _chart.customDomain().includes(d.key0))
-    } else if (chartData) {
-      pieData = chartData
-    }
-
     // if we have data...
-    if (pieData && d3.sum(pieData, _chart.valueAccessor())) {
-      pieData = pie(utils.maybeFormatInfinity(pieData))
+    if (chartData && d3.sum(chartData, _chart.valueAccessor())) {
```

Comment:
Reverting this back to how it was before...do not want to filter data based off immerse-set domain